dts/bindings/misc/nxp,inputmux.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+include: [base.yaml]
+
+compatible: "nxp,inputmux"
+description: NXP i.MX INPUTMUX driver. Routes internal SoC signals.
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: INPUTMUX assignment
+
+  properties:
```

Comment:
would like to see better documentation in this binding, especially more clarity about what the properties mean, how to use the include/dt-bindings file macros, and an example in the main description of the binding being used